guides/nestor/ISPsetup.md

```diff
@@ -0,0 +1,316 @@
+---
+Layout: Post
+Title: ISP setup ubuntu
+Author: Nestor N Torres
+---
+
+### Contact
+- Slack: @nanjuan on wcscusf.slack.com
+- [LinkedIn](https://www.linkedin.com/in/nestor-n-torres-737172a5/)
+- Email: nestor@nntorres.com
+
+### Table
+1. [Prerequisites](#preinfo)
+2. [Summary](#Summary)
+3. [VirtualBox VM Setup](#First)
+4. [Ubuntu Server installation](#Second)
+5. [Ubuntu Server Configuration](#Third)
+
+## Prerequisites <a id="preinfo"></a>
+1. If you have not setup your network adapter look here https://goo.gl/vs4cHC.
```

Comment:
Once you merge your vm setup guide you can link it with the relative path. For now this works but change it once the other guide has been merged.